styles/style.css

```diff
@@ -13,6 +13,187 @@
   background-color: #ffffff;
 }
 
+.top-section {
+  height: 100vh;
```

Comment:
yes, but you need to find the way to avoid this
![image](https://user-images.githubusercontent.com/79144475/200089329-536c3250-5c1e-4421-970b-4cdfb523950c.png)

Reply:
think about min-heigth or combine 2 parameters